code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -325,11 +300,11 @@
 			SSticker.mode.cult+=newstruct.mind
 		SSticker.mode.update_cult_icons_added(newstruct.mind)
 	if(stoner && iswizard(stoner))
-		to_chat(newstruct, "<B>You are still bound to serve your creator, follow [stoner.p_their()] orders and help [stoner.p_them()] complete [stoner.p_their()] goals at all costs.</B>")
+		to_chat(newstruct, "<B>You are still bound to serve your creator, follow their orders and help them complete their goals at all costs.</B>")

```

Comment:
Why this change?

Reply:
It was mostly for the new init_construct proc wasn't passing a 'user' into it because the user could be different from the original master, and I wanted the rest of the 'to_chat' calls to be consistent.  If anyone would like it back to say the person using thing stone's pronoun, then sure.

Reply:
I see. Your explanation makes sense, and I don't have a strong opinion here. Thanks.